symfony/webpack-encore-bundle/1.6/config/packages/test/webpack_encore.yaml

```diff
@@ -0,0 +1,2 @@
+#webpack_encore:
+#    strict_mode: false
```

Comment:
should be removed instead

Reply:
Idea behind leaving this file is to make people aware that there is `strict_mode` option and they don't need to download 1TB of npm modules and compile Webpack just to run functional tests. WDYT?